.github/workflows/ci.yml

```diff
@@ -549,10 +549,12 @@ jobs:
         with:
           toolchain: ${{ env.RUST_VERSION }}
       - uses: Swatinem/rust-cache@v2
+      - name: Install cargo-nextest
+        run: cargo install cargo-nextest
```

Comment:
I think you should use the binstall action instead of compiling it from source. There may also be a GitHub action in the marketplace that ships nextest binaries.

Reply:
Yep good call, will update now.